shell/platform/linux/fl_key_event_plugin.cc

```diff
@@ -173,9 +173,9 @@ static void remove_pending_event(FlKeyEventPlugin* self, uint64_t id) {
   if (self->pending_events->len == 0 ||
       FL_KEY_EVENT_PAIR(g_ptr_array_index(self->pending_events, 0))->id != id) {
     g_warning(
-        "Tried to remove pending event with id %ld, but the event was out of "
+        "Tried to remove pending event with id %ju, but the event was out of "
```

Comment:
I don't think it is necessary to cast the argument. Instead, just use the right format specifier. I believe the cast will be unnecessary if this was %zu or the `PRIu64` macro.

Reply:
>I don't think it is necessary to cast the argument.

If cast is the problem I can rewrite it to use `PRIu64`, It's just for me that using `%ju` is more readably than `PRIu64`.